pom.xml

```diff
@@ -20,19 +20,32 @@
     <modelVersion>4.0.0</modelVersion>
     <parent>
         <groupId>nl.knaw.dans.shared</groupId>
-        <artifactId>dans-prototype</artifactId>
+        <artifactId>dans-scala-prototype</artifactId>
         <version>1.32</version>
     </parent>
     <groupId>nl.knaw.dans.easy</groupId>
     <artifactId>easy-schema</artifactId>
     <name>EASY Schema Definitions</name>
     <description>Collection of metadata schema's used by EASY</description>
     <version>1.x-SNAPSHOT</version>
-    <inceptionYear>2015-2016</inceptionYear>
+    <inceptionYear>2015-2017</inceptionYear>
```

Comment:
agreed. the inceptionYear is described by maven as "The year of the project's inception, specified with 4 digits. This value is used when generating copyright notices as well as being informational." 
http://maven.apache.org/archives/maven-1.x/reference/maven-model/3.0.2/maven.html

so this should be 2015